authors/apps/articles/serializers.py

```diff
@@ -81,14 +83,35 @@ class Meta:
             'image',
             'created_at',
             'updated_at',
-            'tags'
+            'tags',
+            'read_time'
         ]
         read_only_fields = ('slug', 'author',)
 
     def get_author(self, obj):
         serializer = ProfileSerializer(instance=Profile.objects.get(user=obj.author))
         return serializer.data
 
+    def get_read_time(self, obj):
+        """
+        TRT = NoW/275 + [(10/60)*NoI)
+
+        Where;
+        TRT = Total Read Time (in seconds)
+        NoW = Number of Words (in the article)
+        NoI = Number of Images (in the article)
+        """
+        image = obj.image
+        words = obj.body.split()
+        readtime = (len(words)/275.0)*60
+        image_readtime = 10  # Currently we allow a single image (10s)
+
+        if image == "":
+            print(readtime)
```

Comment:
This debug statements should be removed after testing. Even better, you can use `ipdb` for debugging.

Reply:
I'm glad I saw this. Thanks for the tip on `ipdb` @esirK

Reply:
Yeah, I mistakenly pushed the print statements, I prefer using `ipdb` when facing a serious debugging issue or semantic errors. Here I just wanted to see how the data is flowing within the scope of the function before I return it `readtime`. Thanks for this, I have fixed it.